src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -15,7 +15,10 @@
   },
   "stsjorbsmod:AnimateObjects": {
     "NAME": "Animate Objects",
-    "DESCRIPTION": "Deal !D! damage to ALL enemies for each non-exhausted card that didn't start the combat in your deck."
+    "DESCRIPTION": "Deal !D! damage for each non-exhausted card added to your deck this combat to ALL enemies.",
```

Comment:
iryan72 in Discord suggested that they thought `"Deal !D! damage to ALL enemies for each non-exhausted card added to your deck this combat."` read a little more naturally; I agree with that feedback